src/ctim/generate.clj

```diff
@@ -11,6 +11,7 @@
     [campaign :refer [Campaign NewCampaign]]
     [coa :refer [COA NewCOA]]
     [feedback :refer [Feedback NewFeedback]]
+    [identity-assertion :refer [IdentityAssertion NewIdentityAssertion]]
```

Comment:
Could you also add `:identity-assertion` and `:new-identity-assertion` in the `generators`?

The `IdentityAssertion` is also missing in the `ctim.document` ns